### PR TITLE
Resolve #465, #468

### LINKED ISSUE #465: Update sending docs

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: docs/sending.md [hunk 1]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+
+The first step is to choose a wallet from which funds is sent out.
+We support the following three wallets:
+
+> * [Drive Wallet](/wallet)
+> * [Metamask](https://metamask.io/)
+> * [Ledger](https://www.ledger.com/)
+
+Click on the wallet you want to use and proceed to the next step. 
+If you are using a hardware wallet, make sure to have your device plugged.
```
Comment (InVincible2016) [inline docs/sending.md]: make sure to **keep** your device **plugged in/connected**.

File: docs/sending.md [hunk 2]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+
+We autofill the sender email. You only need to provide the following:
+
+> * Recipient Email
+> * Amount
+> * Security Answer (self-provided or auto-generated)
+
+### 4. Review and confirm
+
+Please double check the details with your recipient, such as recipient email address, amount and security answer, and make sure they're correct before confirming your transfer. The transaction fee is also estimated and shown on the review page.
```
Comment (InVincible2016) [inline docs/sending.md]: Please double check the details **of** your recipient, such as **recipient's** email address

File: docs/sending.md [hunk 3]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+
+Click the "Confirm and Transfer" button to continue, and following the instruction at the bottom of the page.
```
Comment (InVincible2016) [inline docs/sending.md]: , and **follow** the instruction at the bottom of the page.

File: docs/sending.md [hunk 4]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+
+## How long does a transfer take?
+
+A single transfer involves two cryptocurrency transactions, one for sending and one for receiving.
+
+The two main factors influencing the transaction time are:
+    1. The amount of network activity
+    2. Transaction fees
+
+We choose the median transaction fee from the last several blocks, assuming a typical daily transaction volume,
```
Comment (InVincible2016) [inline docs/sending.md]: Chainsfer automatically chooses the **median of transaction fees** from the last several blocks

File: docs/sending.md [hunk 5]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+
+> * 15 seconds for ETH, DAI and other ERC20 tokens (1 confirmation)
+> * 60 minutes for BTC (6 confirmations)
+
+In short, 15 seconds for sending out ETH, DAI and other ERC20 tokens, 15 seconds for receiving.
+60 minutes for sending out BTC, and another 60 minutes for receiving. 
+
+## How do you notify me about my transfer?
+
+We will contact you by email when your transfer is created, when your recipient received your funds, and when the transfer expires.
```
Comment (InVincible2016) [inline docs/sending.md]: You will be notified by email when your transfer is created, when your recipient received your funds, and when the transfer expires.

File: docs/sending.md [hunk 6]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+
+> * ~ 0.04 USD for ETH, DAI and other ERC-20 tokens
+> * ~ 1 USD for BTC
+
+
+## Can I send funds to someone with only their email address?
+
+Yes! All you have to do is enter their email address and set a security answer, and we will send them an email with instructions on how to accept the transfer.
+
+## What if I enter my recipient's email wrong?
```
Comment (InVincible2016) [inline docs/sending.md]: What if I **entered** my recipient's email wrong?

File: docs/sending.md [hunk 7]
```diff
@@ -13,6 +14,88 @@ graph LR
...
+## What if I enter my recipient's email wrong? 
+
+We encrypt your transfer data with the security answer provided. Only the one having the correct security answer and the corresponding transfer email link is able
+to receive the funds. The transfer expires after 30 days if not claimed by the recipient. To cancel the transfer, see [Cancellation Section](/cancelling) for details.
+
+## Can I make repeat transfers?
+
+It is not supported currently. Let us know if you think this feature is useful.
+
+## What currencies do you support?
```
Comment (InVincible2016) [inline docs/sending.md]: What currencies does Chainsfer support?

File: docs/sending.md (hunk 6 above)
Comment (timothywangdev) [reply docs/sending.md]: copied from trasferwise, no fix

File: docs/sending.md (hunk 5 above)
Comment (timothywangdev) [reply docs/sending.md]: copied from transferwise, no fix

File: docs/sending.md (hunk 2 above)
Comment (timothywangdev) [reply docs/sending.md]: "Please double check the details with your recipient" copied from trasferwise, no fix

File: docs/sending.md (hunk 7 above)
Comment (timothywangdev) [reply docs/sending.md]: copied from transferwise, no fix

**Pull request merged**